src/main/java/mediathek/config/Konstanten.java

```diff
@@ -20,8 +20,12 @@
 package mediathek.config;
 
 import de.mediathekview.mlib.tool.Functions;
+import de.mediathekview.mlib.tool.Version;
 
 public class Konstanten {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)